### DISCUSSION TITLE
Fresh install errors on "next dev"

### DISCUSSION THREAD

**Discussion by brilang** [vercel/next.js#23335] [Help] [answered]
Having an issue getting Next JS working on a fresh out-of-the-box installation. See screenshots for issue including all commands entered.

![image](https://user-images.githubusercontent.com/867983/112229291-f04db080-8bef-11eb-8fcd-ae4bcdd2abb3.png)
![image](https://user-images.githubusercontent.com/867983/112229313-ff346300-8bef-11eb-8ea3-dcb8ba1930e2.png)

Why doesn't this work out-of-the-box?? 

I am running on Windows 10 Pro
Node 14.16.0
NPM 7.6.3

**Comment by vanyapr** [comment]
Try to run npm -i

**Reply by brilang** [reply]
Added 1 package, but no change on the error message

![image](https://user-images.githubusercontent.com/867983/112229827-ec6e5e00-8bf0-11eb-87ef-e56f97fc9527.png)

**Reply by vanyapr** [reply]
npm i --save @next/env

**Comment by brilang** [comment]
Still getting the same error after that 
![image](https://user-images.githubusercontent.com/867983/112332229-cbe8e700-8c76-11eb-89d6-d14e3e64532a.png)

**Comment by brilang** [comment, marked as answer]
I uninstalled Node & NPM, installed NVM, and started over. Everything worked as expected this time. So something was wrong with my Node install.
